### PR TITLE
SK-1475-WEB-Emails-feature_delegations

### PR DESCRIPTION
**Pull request by BrenKt**
Modification concern 2 guides:

content update for:
pages/web_cloud/email_and_collaborative_solutions/microsoft_exchange/feature_delegation/

Title update for :
pages/web_cloud/email_and_collaborative_solutions/mx_plan/feature_delegation/

### CODE REVIEW

File: pages/web_cloud/email_and_collaborative_solutions/microsoft_exchange/feature_delegation/guide.fr-fr.md [hunk 1]
```diff
@@ -13,6 +13,11 @@ Les services Exchange et Email Pro permettent de bénéficier d’adresses e-mai
 ## Prérequis
 
 - Avoir souscrit une offre [Exchange OVHcloud](https://www.ovhcloud.com/fr/emails/hosted-exchange/) ou [Email Pro](https://www.ovhcloud.com/fr/emails/email-pro/)
+
+> [!warning]
+>
+> **Pour l'offre [Email Pro](https://www.ovhcloud.com/fr/emails/email-pro/)** les fonctionnalités de délégations décritent dans ce guide s'utilisent uniquement au travers du [Webmail](links/web/email). Les protocoles permettant leur utilisation via un logiciel de messagerie nécessite de disposer d'un compte [Exchange](links/web/emails-hosted-exchange).
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/microsoft_exchange/feature_delegation/guide.fr-fr.md]:
```suggestion
> **Pour l'offre [Email Pro](/links/web/email-pro)** les fonctionnalités de délégations décrites dans ce guide s'utilisent uniquement au travers du [Webmail](links/web/email). Les protocoles permettant leur utilisation via un logiciel de messagerie nécessitent de disposer d'un compte [Exchange](links/web/emails-hosted-exchange).
```

File: pages/web_cloud/email_and_collaborative_solutions/microsoft_exchange/feature_delegation/guide.fr-fr.md [hunk 2]
```diff
@@ -128,14 +141,22 @@ Il ne vous reste plus qu'à rédiger votre message et à l'envoyer.
...
 ![delegation](images/delegation-step7.png){.thumbnail}
 
-## Aller plus loin
+## Aller plus loin <a name="go-further"></a>
+
+Pour des prestations spécialisées (référencement, développement, etc), contactez les [partenaires OVHcloud](/links/partner).
+
+Si vous souhaitez bénéficier d'une assistance à l'usage et à la configuration de vos solutions OVHcloud, nous vous proposons de consulter nos différentes [offres de support](/links/support).
 
 Échangez avec notre communauté d'utilisateurs sur <https://community.ovh.com>.
```
Comment (Kazarus4) [inline pages/web_cloud/email_and_collaborative_solutions/microsoft_exchange/feature_delegation/guide.fr-fr.md]:
```suggestion
Échangez avec notre [communauté d'utilisateurs](/links/community).
```

**Review by Kazarus4 — APPROVED** [review]

**Comment by BrenKt** [comment]
Translations to check

File: pages/web_cloud/email_and_collaborative_solutions/microsoft_exchange/feature_delegation/guide.es-es.md [hunk 3]
```diff
@@ -12,9 +12,14 @@ Los servicios Exchange y Email Pro permiten disfrutar de direcciones de correo p
 
 ## Requisitos
 
-- Tener una solución [Exchange](https://www.ovhcloud.com/es-es/emails/hosted-exchange/) o [Email Pro](https://www.ovhcloud.com/es-es/emails/email-pro/) ya configuradas.
+- Tener una solución [Exchange](/links/web/emails-hosted-exchange) o [Email Pro](/links/web/email-pro) ya configuradas.
+
+> [!warning]
+>
+> **Para la oferta [Email Pro](/links/web/email-pro)** las funcionalidades de delegación descritas en esta guía solo se utilizan a través del [Webmail](links/web/email). Los protocolos que permiten su uso a través de un programa de mensajería requieren disponer de una cuenta de [Exchange] (/links/web/emails-hosted-exchange).
```
Comment (Kazarus4) [inline pages/web_cloud/email_and_collaborative_solutions/microsoft_exchange/feature_delegation/guide.es-es.md]:
```suggestion
> **Para la oferta [Email Pro](/links/web/email-pro)** las funcionalidades de delegación descritas en esta guía solo se utilizan a través del [Webmail](links/web/email). Los protocolos que permiten su uso a través de un programa de mensajería requieren disponer de una cuenta de [Exchange](/links/web/emails-hosted-exchange).
```

File: pages/web_cloud/email_and_collaborative_solutions/microsoft_exchange/feature_delegation/guide.es-es.md [hunk 4]
```diff
@@ -23,9 +28,9 @@ Antes de empezar, debe decidir qué permisos quiere delegar. Le recordamos que u
...
-|Permiso de «enviar en nombre de»|Permite realizar un envío en nombre de otra persona. El que aparece como remitente del mensaje es la cuenta que efectivamente realiza el envío, en la que han delegado permisos de «enviar en nombre de». Se incluye una mención indicando que el mensaje se ha enviado en nombre de otra persona.|
-|Permiso de acceso completo|Otorga acceso de solo lectura a la cuenta en la que se ha delegado este permiso. Este acceso permite consultar el contenido, pero no realizar envíos.|
+|**Permiso de «enviar como»**|Permite realizar un envío como si se tratase de otra persona. El que aparece como remitente del mensaje no es la cuenta que realiza el envío, sino la cuenta sobre la que se tienen permisos de «enviar como». El destinatario no sabrá que el mensaje se ha enviado desde otra cuenta, ya que no hay ninguna indicación al respecto.|
```
Comment (Kazarus4) [inline pages/web_cloud/email_and_collaborative_solutions/microsoft_exchange/feature_delegation/guide.es-es.md]:
```suggestion
|**Permiso de "enviar como"**|Permite realizar un envío como si se tratase de otra persona. El que aparece como remitente del mensaje no es la cuenta que realiza el envío, sino la cuenta sobre la que se tienen permisos de "enviar como". El destinatario no sabrá que el mensaje se ha enviado desde otra cuenta, ya que no hay ninguna indicación al respecto.|
```

File: pages/web_cloud/email_and_collaborative_solutions/microsoft_exchange/feature_delegation/guide.es-es.md [hunk 5]
```diff
@@ -23,9 +28,9 @@ Antes de empezar, debe decidir qué permisos quiere delegar. Le recordamos que u
...
+|**Permiso de «enviar en nombre de»**|Permite realizar un envío en nombre de otra persona. El que aparece como remitente del mensaje es la cuenta que efectivamente realiza el envío, en la que han delegado permisos de «enviar en nombre de». Se incluye una mención indicando que el mensaje se ha enviado en nombre de otra persona.|
```
Comment (Kazarus4) [inline pages/web_cloud/email_and_collaborative_solutions/microsoft_exchange/feature_delegation/guide.es-es.md]:
```suggestion
|**Permiso de "enviar en nombre de"**|Permite realizar un envío en nombre de otra persona. El que aparece como remitente del mensaje es la cuenta que efectivamente realiza el envío, en la que han delegado permisos de "enviar en nombre de". Se incluye una mención indicando que el mensaje se ha enviado en nombre de otra persona.|
```

**Review by bs-ovh — APPROVED** [review]

**Review by Kazarus4 — APPROVED** [review]

**Pull request merged**